src/main/groovy/com/adtran/BetablockerPlugin.groovy

```diff
@@ -53,6 +53,11 @@ class BetablockerPlugin implements Plugin<Project> {
                     selection.reject("betablocker rejecting $selection.candidate.displayName (contains $it)")
                 }
             }
+            project.betablocker.rejectedVersionPatterns.each {
+                if (selection.candidate.version.toLowerCase().matches(it)) {
+                    selection.reject("betablocker rejecting $selection.candidate.displayName (matches pattern $it)")
+                }
+            }
```

Comment:
There's a corner case here where a candidate matches both a substring and a pattern (e.g. `1.3.beta0-99273a4`). Granted this is extremely unlikely, but maybe a testcase would be good just to make sure it doesn't throw an exception or something.

Reply:
I've uploaded and artifact with this type of schema. The same tests still pass so :tada: